### PR TITLE
added some FlexNetwork-5130

### PR DESCRIPTION
**Pull request by francesco-noc**

### CODE REVIEW

File: device-types/HPE/FlexNetwork-5130-48G-PoEP-4SFPP-HI.yml [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+manufacturer: HPE
+model: FlexNetwork 5130-48G-POE+-4SFP+-1-slot HI
+slug: hpe-flexnetwork-5130-48G-POEP-4SFPP-HI
+part_number: JH326A
+u_height: 1
+is_full_depth: false
+console-ports:
```
Comment (danner26) [inline device-types/HPE/FlexNetwork-5130-48G-PoEP-4SFPP-HI.yml:8]: Would you be able to add a comment to the datasheet, as well as weight and airflow if you can find that information? Thank you!

File: device-types/HPE/FlexNetwork-5130-48G-4SFPP-HI.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
+manufacturer: HPE
+model: FlexNetwork 5130-48G-4SFP+-1-slot HI
+slug: hpe-flexnetwork-5130-48G-4SFPP-HI
+part_number: JH324A
+u_height: 1
+is_full_depth: false
+console-ports:
```
Comment (danner26) [inline device-types/HPE/FlexNetwork-5130-48G-4SFPP-HI.yml:8]: Would you be able to add a comment to the datasheet, as well as weight and airflow if you can find that information? Thank you!

File: device-types/HPE/FlexNetwork-5130-48G-4SFP+-EI.yml [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+manufacturer: HPE
+model: FlexNetwork 5130-48G-4SFP+ EI
+slug: hpe-flexnetwork-5130-48G-4SFPP-EI
+part_number: JG934A
+u_height: 1
+is_full_depth: false
+comments: '[HPE FlexNetwork 5130 EI Switch Series QuickSpecs](https://www.hpe.com/psnow/doc/c04394228.pdf)'
+console-ports:
```
Comment (danner26) [inline device-types/HPE/FlexNetwork-5130-48G-4SFP+-EI.yml:9]: Would you be able to add weight and airflow if you can find that information? Thank you!

File: device-types/HPE/FlexNetwork-5130-24G-4SFP+-EI.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
+---
+manufacturer: HPE
+model: FlexNetwork 5130-24G-4SFP+ EI
+slug: hpe-flexnetwork-5130-24G-4SFPP-EI
+part_number: JG932A
+u_height: 1
+is_full_depth: false
+comments: '[HPE FlexNetwork 5130 EI Switch Series QuickSpecs](https://www.hpe.com/psnow/doc/c04394228.pdf)'
+console-ports:
```
Comment (danner26) [inline device-types/HPE/FlexNetwork-5130-24G-4SFP+-EI.yml:9]: Would you be able to add weight and airflow if you can find that information? Thank you!

**Review by danner26 — COMMENTED** [review]
Looks like `device-types/HPE/FlexNetwork-5130-24G-4SFP+-EI.yml` still needs weight/airflow if you have it. If not just let me know. Thanks!

**Comment by francesco-noc** [comment]
I don't have airflow.

**Review by danner26 — APPROVED** [review]

**Pull request merged**